### DISCUSSION TITLE
Flexible Bounding Box

### DISCUSSION THREAD

**Discussion by Skywriter960** [osmandapp/OsmAnd#18282] [Ideas]
Hello, 

now I'm writing here as I've already 'bugged' your support enough. 

Yes, have now come across this map app (Android) for bike touring from W10M and have been 'learning the ropes'. Is already extensive - and good!

In the process, a 'problem' has emerged that one does not know from PC-planning: memory!

Can now show my favorite maps as an overlay map according to my wish. It turned out that in the north-south direction I have to download a lot of map material unintentionally. For fun I had then once downloaded my "wish map" and was then in the TB area ... :-(

From the PC I know the flexible bounding box, which can be adjusted to the desired map section that I really need, along with the zoom range.

I would like to see something like this in OSMand in the future. 

Why should I download map material that I do not need, nonsensically occupies my internal memory and also generates unnecessary download costs at the tile server ...

In this sense, thank you for your app!
Skywriter

translate with DeepL

**Comment by DmitryAlexei** [comment]
It is a good proposition. I didn't manage to find similar *Feature request* in the base. So maybe it is worth creating it.

**Comment by DmitryAlexei** [comment]
Or you can write your user story here https://github.com/osmandapp/OsmAnd/issues/14486
